tests/verifier/lib/UDS.pm

```diff
@@ -0,0 +1,69 @@
+##
+# @file UDS.pm
+#
+# @brief Test Suite Unix Domain Socket.
+#
+#
+
+## @class
+# Container for common data.
+package UDS;
+
+use strict;
+use warnings;
+
+
+# Own modules
+use TE::Common;
+use TE::Funclet;
+use TE::Utility;
+
+
+our $test_suite_uds = 
+    [
+{
+            name => 'tc1',
+            note => '#1 - ping-pong w/o arguments',
```

Comment:
it should be UDP test. Is it valid for unix socket.

Reply:
are you referring to the name? the note? both?

Reply:
Is this test is Linux specific or not?

Reply:
Some test are for Linux and some works only for Windows 
e.g feedfile path is different (linux is /tmp/test and windows is different with c:\tmp\test)
another example is AF_UNIX, only work at tcp mode in windows while in linux it works tcp and udp

Reply:
So, udp test is Linux specific, correct and launching verifier under Windows should fail.

Reply:
Yes, but this test run also for windows setup or we have different tests for windows? 
if it's the same- how can I distinguish between the environments?